src/confidential.d.ts

```diff
@@ -1,5 +1,6 @@
 /// <reference types="node" />
 import { Output } from './transaction';
+import type { ZKP as ZKPInterface } from '@vulpemventures/secp256k1-zkp';
```

Comment:
We maybe should revert this if we want to be 100% consistent with bitcoinjs libs. Moreover may be useful to have a global Secp interface defined in liquidjs while using it?

Reply:
What if the interface is stored in LiquidJS? so we leave `@vulpemventures/secp256k1-zkp` and upcoming `@vulpemventures/react-native-secp256k1-zkp` independent to each other

Reply:
Yeah originally I did this, i'm going to restore this interface.